tox.ini

```diff
@@ -2,5 +2,6 @@
 [tox]
 envlist = py27,py34
 [testenv]
-deps=pytest       # install pytest in the venvs
+; install pytest in the venvs
+deps=pytest
```

Comment:
Can you use something like
http://tox.readthedocs.io/en/latest/example/basic.html#compressing-dependency-matrix

to ensure that flakefinder will work with both old / new versions of py.test as well as py2 and py3? At least as it stands, we promise functioning back to 2.7.1. At Dropbox, we're currently using 3.1.3.

Reply:
Did you want to add every major-minor Python version in the matrix?

Reply:
Scratch that. Notice this deprecation warning in the Pytest 3.3.0:
https://docs.pytest.org/en/latest/changelog.html#deprecations-and-removals
> Pytest no longer supports Python 2.6 and 3.3

I assume <2.6 and 3.0-3.3 are deprecated from pytest's POV for 3.3.0+.